src/Stratis.Bitcoin.Features.Apps/AppsFileService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Features.Apps.Interfaces;
+
+namespace Stratis.Bitcoin.Features.Apps
+{
+    public class AppsFileService : IAppsFileService
```

Comment:
You can probably do without the AppsFileService.

We have a class, `Stratis.Bitcoin.Utilities.FileStorage<T>` where you can do 
```
this.fileStorage = new FileStorage<AppConfig>(dataFolder.ApplicationsPath);
IEnumerable<AppConfig> appsConfigs = this.fileStorage.LoadByFileExtension(".json"); // or ".app.json" if you fancy
```

Have a look at how we do it in wallets [here](https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs#L31), [here](https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs#L121) and [here](https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs#L157).

Also, I'm keen for the AppConfig object to be a simple model with attributes that you can just load and use, like in 
```
[JsonProperty(PropertyName = "displayName")]
public string DisplayName { get; set; }

[JsonProperty(PropertyName = "webRoot")]
public string WebRoot{ get; set; }
```
rather than get the fields like so:

```
var displayName = this.appsFileService.GetConfigSetting(fileInfo, "displayName");
var webRoot = this.appsFileService.GetConfigSetting(fileInfo, "webRoot");
```

Reply:
OK, I'll take a look